<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerAstroScalarVolumeDisplayWidget</class>
 <widget class="qSlicerWidget" name="qSlicerAstroScalarVolumeDisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Astro Scalar Volume Display</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="4" column="0" colspan="2">
    <widget class="QWidget" name="PresetsWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="qMRMLWindowLevelWidget" name="MRMLWindowLevelWidget"/>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="WindowLevelPopupButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Push to enable a popup widget to redefine the boundaries of the 2D color function.</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../qSlicerAstroVolumeWidgetsModule.qrc">
           <normaloff>:/Icons/PushPinOut.png</normaloff>:/Icons/PushPinOut.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="LookupTableLabel">
     <property name="text">
      <string>Lookup Table:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qMRMLColorTableComboBox" name="ColorTableComboBox">
     <property name="showChildNodeTypes">
      <bool>true</bool>
     </property>
     <property name="addEnabled">
      <bool>false</bool>
     </property>
     <property name="removeEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="InterpolatePushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Toggle to set/unset the interpolation on 2D views.</string>
       </property>
       <property name="text">
        <string>Slice Interpolation</string>
       </property>
       <property name="icon">
        <iconset resource="../qSlicerAstroVolumeWidgetsModule.qrc">
         <normaloff>:/Icons/SliceInterpolationOn.png</normaloff>:/Icons/SliceInterpolationOn.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ThresholdPushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Toggle to enable widget for the thresholding of the 2D color function.</string>
       </property>
       <property name="text">
        <string>2D Thresholding</string>
       </property>
       <property name="icon">
        <iconset resource="../qSlicerAstroVolumeWidgetsModule.qrc">
         <normaloff>:/Icons/Threshold.png</normaloff>:/Icons/Threshold.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="qMRMLVolumeThresholdWidget" name="MRMLVolumeThresholdWidget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLColorTableComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLColorTableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLVolumeThresholdWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLVolumeThresholdWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWindowLevelWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWindowLevelWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerAstroVolumeWidgetsModule.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>qSlicerAstroScalarVolumeDisplayWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ColorTableComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
